package.json

```diff
@@ -91,6 +91,7 @@
     "silent-error": "^1.0.0",
     "source-map": "^0.5.6",
     "source-map-loader": "^0.2.0",
+    "source-map-support": "^0.4.1",
```

Comment:
Can you add this to https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/package.json as well?

We have several packages being deployed from the same repo so multiple package.json need to be updated.